package-overrides/github/allpro/layout@1.4.3.json

```diff
@@ -9,7 +9,7 @@
     },
     "shim": {
         "jquery-ui-layout": {
-            "deps": "jquery"
+            "deps": ["jquery","jquery-ui"]
```

Comment:
right sorry, misunderstood you the first time, i totally forgot about adding it to dependencies property.
i did not know about the full names in the dependencies. will keep that in mind.